src/mixins/PrintHelper.js

```diff
@@ -1,19 +1,22 @@
 /**
  * A helper mixin to store commonly used functionality
  */
-import { printJob } from '@/api/PrintJobRequests'
 
 const MESSAGE_SUCCESS_PRINTER = 'Printed successfully'
 
 export default {
   name: 'PrintHelper',
   methods: {
     async handlePrintLabel(pipeline, printerName) {
-      let { successful, errors: { message } = {} } = await printJob(
-        printerName,
-        this.selected,
-        pipeline,
-      )
+      const params = {
+        printerName: printerName,
+        barcodesList: this.selected.map((v) => v['barcode']),
+        copies: '1',
+      }
+      const printJobV2Response = await this.printJobV2(params)
+
+      const successful = printJobV2Response.success
```

Comment:
`you can use destructuring as above. const { successful, data: { message } = {} } = await this.printJobV2(params)`

Reply:
OK, had to change successful to success but works otherwise 👍 
I personally think it is less readable, but that's probably just my lack of js familiarity.

Reply:
Maybe. Destructuring is one of the most popular recent additions to JavaScript. I really like it because you can mass assign variables especially those that are deep nested. You can also assign them to new variable names and also assign defaults. It is quite powerful. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment